src/scripts/slurm_submit.sh

```diff
@@ -85,8 +85,17 @@ bls_add_job_wrapper
 ###############################################################
 
 datenow=`date +%Y%m%d`
-jobID=`${slurm_binpath}/sbatch $bls_tmp_file` # actual submission
-retcode=$?
+retry=0
+MAX_RETRY=3
+until [ $retry -eq $MAX_RETRY ] ; do
```

Comment:
Yea, that's 3 tries total but only 2 retries so we should call the variable `MAX_TRIES` or bump the initial value of retry